azure-monitor-opentelemetry-distro/README.md

```diff
@@ -28,6 +28,18 @@ Install the Azure Monitor Opentelemetry Distro with [pip][pip]:
 pip install azure-monitor-opentelemetry-distro --pre
 ```
 
+### Manual Configuration
+
+You can use the `configure_opentelemetry` to set up manual instrumentation in a single line and send telemetry for your app to Azure Monitor. `configure_opentelemetry` supports the following optional arguments:
+
+* connection_string - You can pass in your connections string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 
+* service_name = Specefies the service name. 
+* service_namespace = Specefies the service namespace.
+* service_instance_id = Specefies the instance id.
+* disable_tracing = If set to `True`, disables collection and export or trace telemetry
```

Comment:
The exporter related configurations are missing from here but you can add a TODO and create a tracking issue for this.

Reply:
Added logging_level, sampling_ratio, and export intervals

Reply:
I meant linking to [this](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-opentelemetry-exporter#configuration). Have a section at the bottom saying something like: "Additional configuration related to exporting: "

Reply:
Done